DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,7 +16,9 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="33">
+  <class name="pat::Electron"  ClassVersion="35">
+   <version ClassVersion="35" checksum="1857282274"/>
+   <version ClassVersion="34" checksum="3659401979"/>
```

Comment:
please drop intermediate versions.
33 will need to go to only 34 for this PR alone.